classes/i18n/PKPLocale.inc.php

```diff
@@ -146,6 +146,12 @@ static function initialize($request) {
 		date_default_timezone_set($timeZone);
 
 		if (Config::getVar('general', 'installed')) {
+		    $isSqlServer = Config::getVar('database', 'ms_sql');
+		    if ($isSqlServer) {
```

Comment:
For cleanliness, I'd suggest adding a new `case` below to deal with this. It might cost a few extra cycles in some cases, but it's more compact/consistent.

Reply:
All good, I am gonna add a static function to get the offset from UTC to avoid calling this portion of code each and every time with SQL server.